terraform/examples/README.md

```diff
@@ -0,0 +1,16 @@
+# Terraform Examples
```

Comment:
I would also mention this folder in the `README.md` of the repo where we list subfolders. (And there is a little bug, we should `utm_source=` on one of `README.md` links.

Reply:
Fixed.

`utm_source` is only useful for Mixpanel analytics -- GitHub's repo stats should already should us the referrer if it was an internal link.